packages/core/index.d.ts

```diff
@@ -63,6 +63,7 @@ export interface Bundle<InputData = { [x: string]: any }> {
   targetUrl?: string;
 }
 
+declare class AppError extends Error {}
```

Comment:
`AppError` has a different signature than `Error` with its 3 arguments; that should be reflected here

Reply:
Nice catch - done!